### PR TITLE
Added support for standard Forwarded header

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.7+ |
| Fixed tickets | - |

This PR finishes the work made by @tony-co in #4266.

### CODE REVIEW

File: cookbook/request/load_balancer_reverse_proxy.rst [hunk 1]
```diff
@@ -90,9 +95,12 @@ other information.
 My Reverse Proxy Uses Non-Standard (not X-Forwarded) Headers
 ------------------------------------------------------------
 
-Most reverse proxies store information on specific ``X-Forwarded-*`` headers.
-But if your reverse proxy uses non-standard header names, you can configure
+Although `RFC 7239`_ recently defined a standard ``Forwarded`` header to disclose
+all proxy information, most reverse proxies store information on non-standard
```
Comment (xabbuh) [inline cookbook/request/load_balancer_reverse_proxy.rst]: [...] information in non-standard [...]

Comment (javiereguiluz) [inline cookbook/request/load_balancer_reverse_proxy.rst]: Fixed. Thanks.

File: cookbook/request/load_balancer_reverse_proxy.rst [hunk 2]
```diff
@@ -7,9 +7,14 @@ an AWS Elastic Load Balancer) or a reverse proxy (e.g. Varnish for
...
-header (which will now be the IP address of your reverse proxy), the user's
-true IP will be stored in an ``X-Forwarded-For`` header.
+either the standard ``Forwarded`` header or non-standard special ``X-Forwarded-*``
+headers. For example, instead of reading the ``REMOTE_ADDR`` header (which
+will now be the IP address of your reverse proxy), the user's true IP will be
+stored in a standard ``Forwarded: for="..."`` header or a non standard
+``X-Forwarded-For`` header.
+
+.. versionadded:: 2.7
+    ``Forwarded`` header support was introduced in Symfony 2.7
```
Comment (xabbuh) [inline cookbook/request/load_balancer_reverse_proxy.rst]: missing period at the end of the sentence

Comment (javiereguiluz) [inline cookbook/request/load_balancer_reverse_proxy.rst]: Nice catch. Fixed. Thanks.

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Thanks Javier! Even though this mentions a 2.7 feature, I merged it into the 2.3 branch, because we never actually show any functions that don't exist in 2.3, and because we're mostly talking about something that's universally true (and someone using 2.3 might want to know that the Forwarded header *does* have support in a later version). Cheers!

**Pull request merged**